File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_25.java
import java.util.Scanner;

public class exercicio_25 {

    public static void main(String[] args) {
        int numero, maior = 0;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe um numero: ");
        numero = teclado.nextInt();

        while (numero != 0) {
            if (numero > maior) {
                maior = numero;
            }
            System.out.println("Informe um numero: ");
            numero = teclado.nextInt();
        }

        System.out.printf("O maior numero eh %d", maior);
        teclado.close();
    }

}

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_25.c
#include <stdio.h>

int main() {
    int numero, maior = 0;

    printf("Informe um numero: ");
    scanf("%d", &numero);

    while (numero != 0) {
        if (numero > maior) {
            maior = numero;
        }
        printf("Informe um numero: ");
        scanf("%d", &numero);
    }

    printf("O maior numero eh %d", maior);
}
